### ISSUE TITLE
docker.io openshift/origin-logging-fluentd is missing v3.10.0

### ISSUE THREAD

**Issue by ceagan** [openshift/openshift-ansible#9908]
#### Description

The Ansible scripts refer to version v3.10.0 for many of the Logging dependencies, but only v3.10 is available on Docker Hub. This could be fixed by updating the Ansible scripts or pushing additional tags to Docker Hub.

##### Version

```
ansible 2.4.6.0
openshift-ansible-3.10.41-1-40-g06b17f7
```

##### Steps To Reproduce
1. `ansible-playbook ~/openshift-ansible/playbooks/openshift-logging/config.yml`

##### Expected Results
Successful installation/upgrade is expected.

##### Observed Results
Pull errors occur because referenced images do not exist.

##### Additional Information
Workaround is to add the following variables to inventory file. This forces these variables to be 3.10 instead of 3.10.0.

```
openshift_logging_fluentd_image=openshift/origin-logging-fluentd:v3.10
openshift_logging_elasticsearch_image=openshift/origin-logging-elasticsearch:v3.10
openshift_logging_curator_image=openshift/origin-logging-curator:v3.10
openshift_logging_kibana_image=openshift/origin-logging-kibana:v3.10
openshift_logging_kibana_proxy_image=openshift/origin-logging-auth-proxy:v3.10
```

**Comment by jomeier** [comment]
Same here. It's rather strange why they don't fix it.

**Comment by morsik** [comment]
Went into same problem.

**Comment by mriedmann** [comment]
same with this environment:
```
ansible 2.6.5
openshift-ansible-3.10.53-1-19-g29d0a16
```

A simple fix would pushing a v3.10.0 (in addition to the v3.10) tag to the [docker hub](https://hub.docker.com/r/openshift/origin-logging-fluentd/tags/).

**Comment by rhnewtron** [comment]
How about setting  ?
```
openshift_image_tag=v3.10
```
